pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -336,36 +336,36 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 	if !foundAny {
 		return nil
 	}
-	klog.Warningf("eviction manager: attempting to reclaim %v", resourceToReclaim)
+	klog.InfoS("Eviction manager: attempting to reclaim", "resource", resourceToReclaim)
```

Comment:
again, warning replaced with informational. I will not comment further on this

Reply:
Thak you very much.
I modified it according to the following rules,Help me see if it works?

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md